nucliadb/nucliadb/ingest/orm/node.py

```diff
@@ -54,11 +55,60 @@
 SIDECAR_CONNECTIONS = LRU(50)
 
 
+class NodeType(Enum):
+    IO = 1
+    SEARCH = 2
+    INGEST = 3
+    TRAIN = 4
+    UNKNOWN = 5
+
+    @staticmethod
+    def from_str(label):
+        if label in "Io":
+            return NodeType.IO
+        elif label in "Search":
+            return NodeType.SEARCH
+        elif label in "Ingest":
+            return NodeType.INGEST
+        elif label in "Train":
+            return NodeType.TRAIN
+        else:
+            logger.warn(f"Unknown '{label}' node type")
+            return NodeType.UNKNOWN
+
+    @staticmethod
+    def from_compat(node_type):
```

Comment:
would be nice to add type hints here, so it's clearer from what we are converting to

Reply:
or name it `from_pb`?